### PR TITLE
@craigspaeth => [BUGFIX] Apply button does not jump to "Apply"

### PR DESCRIPTION
**Pull request by damassi**
This PR fixes an issue where the far-right "Apply" button does not jump down to bottom of page.

### CODE REVIEW

File: desktop/apps/partnerships/client/router.coffee [hunk 1]
```diff
@@ -29,3 +30,6 @@ module.exports = class PartnershipsRouter extends Backbone.Router
     $nav = $ '.partnerships-section-nav'
     selector = "##{slug}"
     @jump.scrollToPosition $(selector)?.offset()?.top - $nav.outerHeight() / 2
+
+  toApply: ->
+    @toSection('apply')
```
Comment (damassi) [inline desktop/apps/partnerships/client/router.coffee]: I'm unsure if this "Apply" button ever worked like the other quick-nav items as its structured differently. This seemed to be the most direct fix.

**Comment by craigspaeth** [comment]
Thanks for the quick fix! LGMT, feel free to merge when 💚

**Comment by damassi** [comment]
Will do -- just want to make sure mobile works OK.

**Comment by craigspaeth** [comment]
Thanks for adding the defensive checks! Looks good with the change, feel free to merge when 💚

**Pull request merged**